site/topics/memory/registers-ram.rst

```diff
@@ -193,6 +193,29 @@ Random Access Memory
 Controlling Writes
 ------------------
 
+* When using a decoder to select a memory address, one of the decoded signals is always going to be high
+* This creates a problem as one of the memory addresses would be written to every clock pulse, even if not intended
+* Thus, there needs to be a way to have more control over when data is written to a memory address
+
+* One possible way to address this problem is to have a write enable signal toggle if clock pulses reach the registers
+
+    * This *can* work, but as discussed above, it may create problems in practice
+    * In general, it is best to not manipulate the clock signal
```

Comment:
My brain. Is this duplicated from somewhere, or did I already review this and not approve?

Reply:
You read it before. It's bringing it up again/reminds them of the problem. 


http://modsurski.com/csci263/topics/logic/combinational-logic.html
![image](https://github.com/user-attachments/assets/806f7ada-a314-4094-8679-197d52b24e4e)

Reply:
YOLO